content/en/platform/corda/4.8/open-source/key-concepts-ledger.md

```diff
@@ -20,52 +20,50 @@ title: Ledger
 
 ## Summary
```

Comment:
I think what's missing from this topic is a very general introduction to the concept of a ledger (just a simple one-line intro, such as "A ledger represents the current state of affairs as a journal of transactions.") and a short introduction to the concept of decentralized ledgers. There are some good things in this article from Marco Polo that you could include, that are specific to Corda https://marcopolonetwork.com/articles/distributed-ledger-technology/

I might also include a short note paying lip service to DLT vs blockchain